syndesis/src/modules/data/pages/VirtualizationMetricsPage.tsx

```diff
@@ -1,35 +1,49 @@
 import { RestDataService } from '@syndesis/models';
+import { WithRouteData } from '@syndesis/utils';
 import * as React from 'react';
 import { Translation } from 'react-i18next';
+import HeaderView from '../shared/HeaderView';
 import VirtualizationNavBar from '../shared/VirtualizationNavBar';
 
 /**
  * @param virtualizationId - the ID of the virtualization whose details are being shown by this page.
  */
 export interface IVirtualizationDetailRouteParams {
```

Comment:
Just as an example, `IVirtualizationDetailRouteParams` is present six times (if I counted it correctly) in this PR alone. I would suggest renaming it to be `IVirtualizationMetricsPageRouteParams` instead, so to tie it to this specific route.

Reply:
Corrected :)